Sources/Fullscreen/NotificationCenter/ActivityIndicatorSectionFooterView.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  ActivityIndicatorSectionFooterView.swift
+//  FinniversKit
+//
+//  Created by Granheim Brustad , Henrik on 04/02/2020.
+//  Copyright © 2020 FINN AS. All rights reserved.
+//
+
+import Foundation
+
+class ActivityIndicatorSectionFooterView: UITableViewHeaderFooterView {
+
+    let activityIndicatorView: UIActivityIndicatorView = {
+        if #available(iOS 13, *) {
+            return UIActivityIndicatorView(style: .medium)
+        } else {
+            return UIActivityIndicatorView(style: .gray)
+        }
+    }()
+
+    override init(reuseIdentifier: String?) {
+        super.init(reuseIdentifier: reuseIdentifier)
+
+        activityIndicatorView.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
I guess to follow the common pattern you should move the stuff here to `setup()`, but no big deal on such a small class